inst/csv/replacementPatterns.csv

```diff
@@ -247,4 +247,50 @@ From,To,Pattern,Replacement
 "sql server","impala",".dbo.","."
 "sql server","impala","##","<hash><hash>"
 "sql server","impala","#","%temp_prefix%%session_id%"
-"sql server","impala","<hash><hash>","##"
\ No newline at end of file
+"sql server","impala","<hash><hash>","##"

```

Comment:
I don't understand the translation from <hash><hash> to ##. <hash><hash> doesn't exist as a valid staetment in sql server, so we'd never see that to translate from sql to impala.